script.js

```diff
@@ -1,43 +1,48 @@
 class Books {
-    constructor(title, author) {
-        this.title = title;
-        this.author = author;
-        this.BookId = this.BookId;
-    }
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
 
 const bookArchive = [];
-const form  = document.querySelector('.form');
+const form = document.querySelector('.form');
 const booksDiv = document.querySelector('.books');
 
-
-
+function addNewBook(book) {
+  const bookUnit = document.createElement('li');
+  bookUnit.innerHTML = `
+      <p class="book-name">${book.title}</p>
+      <p class="the-auhtor">${book.author}</p>
+      <button class='removeBook' >Remove</button>
+      `;
+  booksDiv.appendChild(bookUnit);
+}
 
 form.addEventListener('submit', (event) => {
-    event.preventDefault();
-
-    const title = document.querySelector('.book-title').value;
-    const author = document.querySelector('.book-author').value;
+  event.preventDefault();
 
-    const book = new Books (title, author)
-    bookArchive.push(book);
-    addNewBook(book)
+  const titleInput = document.querySelector('.book-title');
+  const authorInput = document.querySelector('.book-author');
+  const title = document.querySelector('.book-title').value;
+  const author = document.querySelector('.book-author').value;
+  const book = new Books(title, author);
 
-})
-
-
-function addNewBook(book) {
-    booksDiv.innerHTML = `
-    <p class="book-name">${book.title}</p>
-    <p class="the-auhtor">${book.author}</p>
-    <button class='removeBook' >Remove</button>
-    `
-}
+  bookArchive.push(book);
+  addNewBook(book);
 
-// const removeButton = document.querySelector('.removeBook');
-// console.log(removeButton)
+  localStorage.setItem('bookInfo', JSON.stringify(bookArchive));
 
```

Comment:
- [ ] [REQUIRED] Good job so far! Data should be preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md), reloading the page manually to check this after `adding` a book and `removing` a book. Date is being saved to local storage but it is not rendering to display, please fix it 🙏🏼 

![ezgif-4-831aa3158c](https://user-images.githubusercontent.com/63915024/212915802-3bf81ed0-5f0b-416e-8776-559b0300e33f.gif)